php/php.api.phpmodule/manifest.mf

```diff
@@ -1,4 +1,4 @@
 Manifest-Version: 1.0
 OpenIDE-Module: org.netbeans.modules.php.api.phpmodule
 OpenIDE-Module-Localizing-Bundle: org/netbeans/modules/php/api/phpmodule/resources/Bundle.properties
-OpenIDE-Module-Specification-Version: 2.71
+OpenIDE-Module-Specification-Version: 2.72
```

Comment:
Don't we want to increase the dependency version in `php.kit`?

Reply:
It seems that `php.kit` doesn't have `phpmodule.api`as module-dependencies: https://github.com/apache/netbeans/blob/master/php/php.kit/nbproject/project.xml

Reply:
OK, so some other module has to depend on it (my bet is `php.project`). But that would mean that also version of that module (together with `php.kit`) would need to be increased - probably not needed (I have no idea how "live" NB updates are). So, let's keep it as it is, thanks.

Reply:
> OK, so some other module has to depend on it (my bet is `php.project`).

Yes, `php.project`, `php.editor`, etc.

> So, let's keep it as it is

OK.